eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -164,7 +169,11 @@ def params(self):
             offset = (int)(random.random() * math.fabs(t2 - t1))
 
             ts_max_ms = int(offset + min(t1, t2))
-        
+
+        if self._window_length == "random":
+            max_window_length = int(math.fabs(self._fieldstats_end_ms - self._fieldstats_start_ms))
```

Comment:
Should we just use `abs` here instead of `math.fabs` as we convert the result anyway with `int(...)`? I see that this class uses `math.fabs` elsewhere so maybe we want to stay consistent and just keep it as is. I'm fine either way.

Reply:
I'm fine with either, mainly used to stay consistent